src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -394,24 +396,43 @@
     end
   end
 
+  describe 'POST #set_bugowner_request' do
+    let(:req) { BsRequest.find_by(creator: submitter_with_group.login, description: 'blah blah blash', state: 'new') }
```

Comment:
Please rename that to request. It's better (== easier to understand) to use speaking / selfexplaing variable names.

Reply:
Nope, unfortunatelly ```request``` is already used by rspec as ```response```, since in the controller we are using it as ```req``` I think it makes sense to let it as it is. I hope it's ok.

Reply:
Given that you introduce this here, in the test file, I would change it here at least and rename it as bs_request.